spotifycli/spotifycli.py

```diff
@@ -98,106 +129,190 @@ def get_arguments():
         ("--playpause", "plays or pauses the song (toggles a state)"),
         ("--lyrics", "shows the lyrics for the song"),
         ("--next", "plays the next song"),
-        ("--prev", "plays the previous song")
+        ("--prev", "plays the previous song"),
+        ("--export", "export lyrics"),
     ]
 
 
+def set_genius_token(token):
+    os.environ[TOKEN_ENV_VAR] = token
+
+    with open(os.path.join(expanduser("~"), TOKEN_FILE_NAME), "w") as f:
+        f.write(token)
+
+    print("Token was set")
+
+
+def load_genius_token():
+    file_path = os.path.join(expanduser("~"), TOKEN_FILE_NAME)
+
+    if os.path.isfile(file_path):
+        with open(os.path.join(expanduser("~"), TOKEN_FILE_NAME), "r") as f:
+            os.environ[TOKEN_ENV_VAR] = f.read()
+    else:
+        print(
+            "No genius api token was set. Some lyrics will not be found. "
+            "Use --token option to set it!"
```

Comment:
This message is displayed all the time by the program even if someone doesn't want to use this token or API!